packages/model/src/news.ts

```diff
@@ -1,6 +1,11 @@
 import { ListResponse } from './common';
 
-export const newsType = ['News', 'Event', 'Training'] as const;
+/* istanbul ignore next */ const newsType = [
```

Comment:
This was flagging on code coverage. Honestly testing an array works feels weird. Not sure why it complained about this so i've ignored it. Happy to take suggestions on this